getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Events"
+weight: 20
+---
+
+# Events
+
+
+### Introduction
+
+Events are portions of data which can be sent to a chat ([send_event](../../agent-api/client-server#send-event)). There are different types of events:
+
+ - [message](#message)
+ - [annotation](#annotation)
+ - [filled-form](#filled-form)
+ - [system message](#system-message)
+ - [file](#file)
+ - [custom event](#custom-event)
+
+
+[Incoming event](../../agent-api/server-client#incoming-event) push will inform you about events sent to a chat (on both, agent and customer side).
+
+
+### Common event fields
+
+All events have some common fields, we will describe them here to avoid duplication:
+
+| name        | puprose                                        | filled by | type   |
+|-------------|------------------------------------------------|-----------|--------|
+| `id`        | id of the event                                | server    | string |
+| `custom_id` | is an ID that you can set for your own purpose | client    | string |
+| `order`     | event with lower `order` are hronologicaly before | server | int    |
+| `author_id` | this is id of sender of the event              | server    | string |
+| `text`      | this is payload that you can send as event (in example message content) | client | string |
```

Comment:
this is payload that you can send as event (in example message content) -> this is the payload that you can send as within an event (e.g. message content)